platforms/hermes-javascript/scripts/postinstall.js

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+
+const os = require('os')
+const fs = require('fs')
+const readline = require('readline')
+const wretch = require('wretch').default
+
+const { logger, osIsRaspbian, LIB_EXTENSION, LIB_DIST } = require('./utils')
+const hermesMqttVersion = require('../package.json')['hermes-mqtt-version']
+
+const request = wretch(`http://s3.amazonaws.com/snips/hermes-mqtt/${hermesMqttVersion}`).polyfills({
+    fetch: require('node-fetch')
+})
+
+const OS_SUPPORTED = [
+    'linux',
+    'darwin'
+]
+
+const ARCHITECTURES_SUPPORTED = [
+    'arm',
+    'x64'
+]
+
+function getPlatformName () {
+    // Linux or Darwin or Windows_NT
+    const osType = os.type().toLowerCase()
+    // 'arm', 'arm64', 'ia32', 'mips', 'mipsel', 'ppc', 'ppc64', 's390', 's390x', 'x32', 'x64'.
+    const architecture = os.arch()
+    // os version
+    const osRelease = os.release()
+    // cpu data
+    const cpuInfo = os.cpus()[0]
+
+    if (OS_SUPPORTED.indexOf(osType) < 0) {
+        logger.warning(`There is no prebuilt dynamic library file available for your operating system (${osType}).\n`)
+        return false
+    }
+    if (ARCHITECTURES_SUPPORTED.indexOf(architecture) < 0) {
+        logger.warning(`There is no prebuilt dynamic library file available for your hardware architecture (${architecture}).\n`)
+        return false
+    }
+
+    // Minimum osx version supported is El Capitan (15.0.0)
+    if(osType === 'darwin') {
+        if((+osRelease.split('.')[0]) < 15) {
+            logger.warning(`There is no prebuilt dynamic library file available for your version os MacOS (${osRelease}).\n`)
+            return false
+        }
+        return 'macos-darwin-x86_64'
+    }
+
+    if(osIsRaspbian()) {
+        // Support only ARMv7
+        if(cpuInfo.model.toLowerCase().indexOf('armv7') < 0) {
+            logger.warning('Prebuilt dynamic library file is only available for ARMv7 cpus.\n')
```

Comment:
hum, this should work on a pi zero armv6 (ought to be tested though but we use the `arm-unknown-linux-gnueabihf`  rust target and not `armv7-unknown-linux-gnueabihf` so we should be fine)

Reply:
relaxed the check to make it work on raspbian + arm6/7